src/languages/es.js

```diff
@@ -172,13 +172,10 @@ export default {
     },
     reportActionsView: {
         beginningOfChatHistory: 'Aquí comienza tu historial de conversaciones con ',
-        beginningOfChatHistoryPrivatePartOne: 'Este es el principio de la sala privada ',
-        beginningOfChatHistoryRestrictedPartOne: 'Este es el principio de ',
-        beginningOfChatHistoryPrivatePartTwo: ', invita a otros @mencionándolos.',
-        beginningOfChatHistoryRestrictedPartTwo: ', invita a otros @mencionándolos.',
-        beginningOfChatHistoryPolicyExpenseChatPartOne: '¡La colaboración entre ',
-        beginningOfChatHistoryPolicyExpenseChatPartTwo: ' y ',
-        beginningOfChatHistoryPolicyExpenseChatPartThree: ' empieza aquí! :tada: Este es el lugar donde chatear, pedir dinero y pagar.',
+        beginningOfChatHistoryPrivate: ({reportName}) => `Este es el principio de la sala privada <strong>${reportName}</strong>, invita a otros @mencionándolos.`,
+        beginningOfChatHistoryRestricted: ({reportName}) => `Este es el principio de <strong>${reportName}</strong>, invita a otros @mencionándolos.`,
+        beginningOfChatHistoryPolicyExpenseChat: ({owner, policyName}) => `¡La colaboración entre <strong>${owner}</strong> y <strong>${policyName}</strong> empieza aquí! :tada: Este es el lugar donde chatear, pedir dinero y pagar.`,
+        beginningOfChatHistoryDeleted: ({reportName}) => `Este es el principio de <strong>${reportName}</strong>, no hay nada que ver aquí.`,
```

Comment:
Need to verify if this is the copy we wanna use and need to verify the translation of the copy, will ask about it on the channel (on monday).

Reply:
@roryabraham please help us with adding the `waiting for copy` label. Thanks!